### ISSUE TITLE
Wired Deathadder Essential is not being picked up, Fedora 35

### ISSUE THREAD

**Issue by mohammadsav** [openrazer/razer-drivers#1794]
As of today I have updated the relavent files from openrazer and tried using RazerGenie but the mouse would not be detected. The app definitely worked because it showed me "No Device Was Connected"

**Comment by Etaash-mathamsetty** [comment]
can you dump the output of lsusb?

**Comment by mohammadsav** [comment]
[lsusb_verbose_output.zip](https://github.com/openrazer/openrazer/files/8459053/lsusb_verbose_output.zip)

**Comment by mohammadsav** [comment]
Btw I was using polychromatic and just this is what outputs when I clicked the troubleshoot button. https://imgur.com/a/ecEJLnZ

**Comment by mohammadsav** [comment]
Sorry one last thing, I tried using this for both Razergenie and Polychromatic. Both applications were functioning, and both did not recognize my mouse.

**Comment by Etaash-mathamsetty** [comment]
`Bus 001 Device 002: ID 1532:0098 Razer USA, Ltd Razer DeathAdder Essential`
so your mouse is being picked up by linux.
What version of openrazer are you using, support for this mouse was added in v3.2 (I have this mouse too actually)

**Comment by Etaash-mathamsetty** [comment]
oh I didn't look at that image, you have to modprobe the drivers, so open terminal, and run
`sudo modprobe razermouse`
and post your output (if any)

**Comment by mohammadsav** [comment]
I think I solved it. I ran the "sudo modprobe razermouse" and copy pasted the output in google and found a comment on support post  [#98](https://github.com/openrazer/openrazer/issues/98#issuecomment-271167846) 
Both RazerGenie and Polychromatic work now for my Fedora 35.

Just for any future people, what outputed before fixing it was this https://imgur.com/a/HlnYiGT
The solution was to disable secure boot.

Thank you for taking the time to help me @Etaash-mathamsetty 

edit: grammar
